### PR TITLE
Weekly CLI backmerge

### PR DESCRIPTION
**Pull request by technophile-04**
## Description

- Track hardhat-deploy deployments, except localhost (#666)
- feat: add external flag to external contracts (#647)

tested it nicely and works great 🙌

Note: We shouldn't squash & merge this instead do a normal merge

### CODE REVIEW

File: templates/base/packages/nextjs/utils/scaffold-eth/contractsData.ts [hunk 1]
```diff
@@ -0,0 +1,8 @@
+import scaffoldConfig from "~~/scaffold.config";
+import { contracts } from "~~/utils/scaffold-eth/contract";
+
+export function getAllContracts() {
+  const contractsData = contracts?.[scaffoldConfig.targetNetworks[0].id];
+  console.log("The contracts data is:", contractsData);
```
Comment (rin-st) [inline templates/base/packages/nextjs/utils/scaffold-eth/contractsData.ts:6]: is it needed or someone forgot to remove it?

**Review by rin-st — APPROVED** [review]
🙌

**Review by Pabl0cks — APPROVED** [review]
Both features working nicely on my fresh CLI generated project. GJ Shiv!! 🙌🙌

**Comment by technophile-04** [comment]
Thanks all 🙌 Just removed `.github/ISSUE_TEMPLATE` and pr template as commented https://github.com/scaffold-eth/scaffold-eth-2/pull/661#issuecomment-1856042964 and they were kind of specific to SE-2

**Pull request merged**